dimod/binary/binary_quadratic_model.py

```diff
@@ -1823,13 +1823,37 @@ class Float64BQM(BQM, default_dtype=np.float64):
     pass
 
 
-def Binary(label: Variable, bias: Bias = 1,
+@unique_variable_labels
+def Binary(label: Optional[Variable] = None, bias: Bias = 1,
            dtype: Optional[DTypeLike] = None) -> BinaryQuadraticModel:
+    """Return a :class:`.BinaryQuadraticModel` instance representing a binary variable.
```

Comment:
`Return a binary quadratic model with a single variable.`

Reply:
We don't normally use the reference links in the first line of the docstring, sphinx doesn't really know what to do with them.